pom.xml

```diff
@@ -1455,7 +1462,7 @@
     <xml.apis.version>1.4.01</xml.apis.version>
     <jstl.version>1.2</jstl.version>
     <openid4java.version>0.9.8</openid4java.version>
-    <cas-server-security-filter.version>2.0.0</cas-server-security-filter.version>
+    <cas-server-security-filter.version>2.0.1-SNAPSHOT</cas-server-security-filter.version>
```

Comment:
Forget my comment, I missed your latest PR on the `cas-server-security-filter`...